src/Tools/Traits/ParamHelpers.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Mpociot\ApiDoc\Tools\Traits;
+
+trait ParamHelpers {
+
+    /**
+     * Create proper arrays from dot-noted parameter names
+     *
+     * @param array $params
+     * @return array
+     */
+    protected function cleanParams(array $params)
+    {
+        $values = [];
+        foreach ($params as $name => $description) {
```

Comment:
Having `$description['value']` is confusing. Let's name this `$details` or something else.
```suggestion
        foreach ($params as $name => $details) {
```